style.css

```diff
@@ -0,0 +1,74 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');
+
+body {
+  margin: 0;
+  padding: 0;
+  background-color: #e5e5e5;
+}
+
+.navbar {
+  padding: 5px 25px;
+  height: 48px;
+  background-color: white;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav-btn {
+  border: none;
+  background-color: transparent;
+  font-size: 10px;
+  color: #6070ff;
+}
+
+.logo h3 {
+  font-family: "Poppins", sans-serif;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  margin: 0;
+  color: #6070ff;
+}
+
+.info {
+  padding: 140px 22px 60px 22px;
+  background-image: url("images/header-shapes\ mobile.svg");
```

Comment:
- [ ] Kindly change the background image's `background-repeat` property  to `no-repeat`. Check about background image properties through [this](https://www.w3schools.com/cssref/pr_background-image.php) and [this](https://developer.mozilla.org/en-US/docs/Web/CSS/background-image) links.

- Your design
<img width="289" alt="wrong" src="https://user-images.githubusercontent.com/41163571/227461288-c9591e89-c20b-478d-bb32-d6c60522d6b5.png">